### PR TITLE
Reduce rules line weight

### PR DESCRIPTION
**Pull request by alberto-lazari**
The first thing I noticed about the proof trees is that the line weight of the rules are thicker than the ones in bussproofs. They don't look right to me, because `1pt` for the stroke weight is too much. I reduced it to `0.5pt` which looks to be the same of bussproof's

Alternatively you should put it as a parameter of `prooftree()`, but it's not that convenient, because the function doesn't support `#show` and `#set` rules, so the tree can't be customized for the entire document

### CODE REVIEW

**Comment by SkiFire13** [comment]
I'm ok with changing the default, but it would also be nice to have a parameter on `prooftree`. You can "set" it for the current scope if you do something like `let prooftree = prooftree.with(line-stroke: 42pt)`

File: prooftree.typ [hunk 1]
```diff
@@ -10,9 +10,10 @@
     side: left,
     padding: 0.2em,
   ),
+  line-stroke: 0.5pt,
   ..rules
 ) = {
-  // Check the types of `spacing` and `label`.
+  // Check the types of the parameters.
```
Comment (alberto-lazari) [inline prooftree.typ:16]: Okay, maybe it's not exactly true that we're checking every parameter, but only the "first level" ones

File: prooftree.typ [hunk 2]
```diff
@@ -23,6 +24,11 @@
     message: "The value `" + repr(label) + "of the `label` argument was expected"
       + "to have type `dictionary` but instead had type `" + type(label) + "`."
   )
+  assert(
+    type(line-stroke) == "length",
+    message: "The value `" + repr(line-stroke) + "of the `line-stroke` argument was expected"
```
Comment (SkiFire13) [inline prooftree.typ]:
```suggestion
    message: "The value `" + repr(line-stroke) + "` of the `line-stroke` argument was expected"
```

File: prooftree.typ [hunk 3]
```diff
@@ -86,6 +92,7 @@
       side: label.at("side", default: left),
       padding: label.at("padding", default: 0.2em),
     ),
+    line-stroke: 0.5pt,
```
Comment (SkiFire13) [inline prooftree.typ]:
```suggestion
    line-stroke: line-stroke,
```

We don't want to ignore the parameter passed by the user :)

**Review by SkiFire13 — CHANGES_REQUESTED** [review]

File: prooftree.typ (hunk 1 above)
Comment (SkiFire13) [reply prooftree.typ:16]: I just wanted to avoid someone looking at these asserts and wondering wtf is going on, so this is fine.

**Pull request merged**